src/gsi/m_berror_stats_reg.f90

```diff
@@ -1132,11 +1167,141 @@ subroutine read_howv_stats(nlat,nlon,npar,arrout)
             read(unit=lun34 ,rec=j) (arrout(i,j,i_npar), i=1,nlat)
          enddo
          close(unit=lun34)
+         if (print_verbose .and. mype .eq. 0) then         
+           print*,trim(adjustl(myname)), trim(filename(i_npar)) // ' is used for background error of howv.'
+         end if
 
       else 
-         print*,myname, trim(filename(i_npar)) // ' does not exist'
+         if (print_verbose .and. mype .eq. 0) then         
+           print*,trim(adjustl(myname)), trim(filename(i_npar)) // ' does not exist for static BE of howv.'
+           print*,trim(adjustl(myname)), '::using pre-defined corp_howv=',corp_howv, ' hwllp_howv=',hwllp_howv, ' for howv.'
+         end if
       end if
    end do
 end subroutine read_howv_stats
+!++++
```

Comment:
I am not familiar with current HOWV DA in 2D RTMA and hence have a general question: in operational 2D RTMA, do we read corp/hwllp from a binary file and we use a default value of 0.42/50km?

Reply:
As what I learned from Manuel, operational 2D RTMA does read corp and hwllp from two binary format files actually. This was originally designed by Stelios and Manuel for 2DRTMA. And the default value 0.42 meters (for corp) and 50 km (for hwllp) are used for 2DRTMA if the binary files are not available for 2DRTMA. These are legacy code.
Manuel thinks 50km might be short for HOWV analysis in 3DRTMA, so he suggests 170km as default for 3DRTMA (which is initialized in rapidrefresh_cldsurf_mod.f90).